python/py-doxyqml/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               python 1.0
+
+name                    py-doxyqml
+version                 0.5.3
+categories-append       textproc
+platforms               {darwin any}
+supported_archs         noarch
+license                 BSD
+maintainers             {@RJVB gmail.com:rjvbertin} openmaintainer
```

Comment:
don't add someone else as maintainer unless explicitly requested/approved by them.

Reply:
The Portfile submitted on the Trac ticket had that individual as a maintainer. I'll remove it, but that was the reason why I added it afterwards; I simply missed it when originally pushing.